action.yml

```diff
@@ -13,16 +13,22 @@ outputs:
   tag:
     description: "Tag, if one was created"
     value: ${{ steps.run.outputs.tag }}
+  version:
+    description: "Version read from package-yaml"
+    value: ${{ steps.prep.outputs.result }}
 runs:
   using: composite
   steps:
+    - id: prep
+      uses: mikefarah/yq@v4.14.1
+      with:
+        cmd: yq eval '.version' '${{ inputs.package-yaml }}'
+
     - id: run
       name: Run
       shell: bash
-      working-directory: ${{ inputs.working-directory }}
       run: |
         ${{ github.action_path }}/bin/run \
-          -y "${{ inputs.package-yaml }}" \
-          -p "${{ inputs.tag-prefix }}" \
           -r "${{ github.repository }}" \
-          -c "${{ github.sha }}"
+          -c "${{ github.sha }}" \
+          -t "${{ inputs.tag-prefix }}${{ steps.prep.outputs.result }}"
```

Comment:
This reduces `bin/run` to basically "create tag if not exists". I had hoped to find an out-of-the-box action that just did that, that would make the whole thing a wildly-simple two-step composite action.

There are many "create tag" action, but they all automatically decide what tag to create (i.e. semantic commit stuff), generate changelogs, etc, and can't be convinced to just create some explicit tag. I did find one action that sort of fit the bill, but it was almost identical to `bin/run` (surprise), so I figured we might as well keep our thing.